packages/react-composites/tests/browser/call/hermetic/VerticalGalleryResizing.test.ts

```diff
@@ -87,7 +87,7 @@ test.describe('Vertical gallery resizing tests', async () => {
     await waitForSelector(page, dataUiId(IDS.verticalGalleryVideoTile));
 
     // check initial tile number to be correct.
-    expect(await page.locator(dataUiId(IDS.verticalGalleryVideoTile)).count()).toBe(2);
+    await waitForSelectorCount(page, dataUiId(IDS.verticalGalleryVideoTile), 2);
```

Comment:
nope, it waits for the correct count and fails if the condition isn't met. Change here allows two things:
- the count is now waited for (so can be in subsequent render passes)
- the action will now screenshot on a failure

Reply:
So previous one will be waiting for dataUiId(IDS.verticalGalleryVideoTile), if that is showing up, it will start counting the number

But the current behavior would be wait until it is 2 count, and failed it that is not (and take screenshot I guess?)

Reply:
I am good with this one, this might miss some "delay render"(like first second it renders one element, but later render 2 elements which meet the condition) issues, but also allow use to get more stable UI snapshot

It could have either make our pipeline faster or slower, would like to see the final number when we have this in main

Reply:
I believe the previous one actually did no waiting, it just immediately checked what was rendered. We now instead wait for the condition to become true because the resize observer kicks in after mounting so the tiles we show changes across a couple render passes